services/src/main/java/io/grpc/services/BinlogHelper.java

```diff
@@ -548,19 +538,21 @@ static Peer socketToProto(SocketAddress address) {
   }
 
   @VisibleForTesting
-  static io.grpc.binarylog.Metadata metadataToProto(Metadata metadata, int maxHeaderBytes) {
-    Preconditions.checkNotNull(metadata);
-    Preconditions.checkState(maxHeaderBytes >= 0);
-    Builder builder = io.grpc.binarylog.Metadata.newBuilder();
+  static void metadataToProto(
+      GrpcLogEntry.Builder entryBuilder, Metadata metadata, int maxHeaderBytes) {
```

Comment:
nit: might consider naming this something like addMetadataToProto

Reply:
renamed